packages/evm/contracts/adapters/Hyperbridge/HyperbridgeAdapter.sol

```diff
@@ -10,6 +10,17 @@ contract HyperbridgeAdapter is BlockHashAdapter, Ownable, BaseIsmpModule {
     mapping(bytes32 => bytes32) public enabledReporters;
     mapping(bytes32 => uint256) public chainIds;
 
+    // @dev The address of the IsmpHost on the current chain
+    address public _host;
```

Comment:
variable with _` `before the name should be private. If you want to keep it public please remove it.

Reply:
Slight oversight